support/Pipelines/Bbh/PostprocessId.py

```diff
@@ -111,22 +116,36 @@ def postprocess_id(
     for object_label, xcoord, excision_radius in zip(
         ["AhA", "AhB"], [x_A, x_B], [excision_radius_A, excision_radius_B]
     ):
-        _, horizon_quantities = find_horizon(
-            id_volfiles,
-            subfile_name=id_subfile_name,
-            obs_id=obs_id,
-            obs_time=0.0,
-            initial_guess=Strahlkorper[Frame.Inertial](
+        horizon_quantities = None
+        if not negative_expansion_bc:
```

Comment:
Generally prefer positive conditions because they are easier to understand (somehow it takes a lot of brain cycles to negate a condition). Just do `if negative_expansion_bc` first.